src/utils/attribution-html.js

```diff
@@ -32,4 +32,29 @@ function getAttributionHtml(media, licenseUrl, fullLicenseName) {
   return `<p style="font-size: 0.9rem;font-style: italic;">${mediaTag}${imgLink}${creator}${licenseLink}${licenseImgLink}</p>`
 }
 
+/**
+ * Get the HTML markup for attributing the given media item.
+ *
+ * @param mediaItem - the media item, image or audio from the API, to attribute
+ * @returns {string} the HTML markup of the attribution
+ */
+export const getAttributionMarkup = (mediaItem) => {
+  if (!mediaItem) return ''
+
+  let title = `"${mediaItem.title}"`
+  if (mediaItem.foreign_landing_url) {
+    title = `<a href="${mediaItem.foreign_landing_url}">${title}</a>`
+  }
+
+  let creator = mediaItem.creator
+  if (mediaItem.creator_url) {
+    creator = `<a href="${mediaItem.creator_url}">${creator}</a>`
+  }
+
+  let license = `${mediaItem.license} ${mediaItem.license_version}`
+  license = `<a href="${mediaItem.license_url}">${license.toUpperCase()}</a>`
+
+  return `${title} by ${creator} is licensed under ${license}.`
+}
```

Comment:
We probably want to use `target="_blank"` and `rel="noopener noreferrer"` on these links. The added complexity might warrant a small helper, something similar to this:

```ts
const buildAnchorTag = (text, href) => 
  `<a target="_blank" rel="noopener noreferrer" href=${href}>${text}</a>`
```

Reply:
What are CC's policies on attribution language? Can you attribute in any language? Are there any rules that the atribution must be understood by the viewers? If I run a blog in English and my audience is largely English-speaking, do I necessarily have to attribute media I use in English?